docs/pywbemclisubcommands.rst

```diff
@@ -45,6 +45,190 @@ Command Details`
+  ``-o`` or ``--name-only`` defines whether only the instance name or the complete
+  object will be displayed.
+
+* The ability to interactively select the data for certain objects as opposed
+  to having to enter the full name.
+
+
+.. _`Using pywbem Pull Operations from pywbemcli`:
+
+Using pywbem Pull Operations from pywbemcli
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Pywbem includes multiple ways to execute the enumerate instance type operations
+(``Associators``, ``References``,`` EnumerateInstances``, ``ExecQuery``):
+
+* The traditional operations (ex. ``EnumerateInstances``)
+* The pull operations (ex. the pull sequence ``OpenEnumerateInstances``, etc.)
+* An overlay of the above two operations called the ``Iter..``. operations where
+  each ``Iter..`` operation executes either the traditional or pull operation
+  depending on a parameter of the connection.
+
+Pywbemcli implements these method calls to pywbem using the ``Iter...``
+operations so that either pull operations or traditional operation can be used
+simply by changing a pywbemcli input parameter (``--use-pull-ops``).
+
+Two options on the command line allow the user to either force the use of pull
+operations, of traditional operations, or to let pywbem try them both.
+
+The input parameter ``--use-pull-ops`` allows the choice of pull or traditional
+operations with the default being to allow pywbem to decide.  The input
+parameter ``max_object_cnt`` sets the ``MaxObjectCount`` variable on the operation
+request if the pull operations are to be used which tells the WBEM server to
+limit the size of the response.  For example::
+
+    pywbemcli --server http/localhost use-pull-ops=yes max_object_cnt=10
+
+would force the use of the pull operations and return an error if the target
+server did not implement them and would set the ``MaxObjectCount`` parameter on the
+api to 10, telling the server that a maximum of 10 objects can be returned for
+each of the requests in an enumeration sequence.
+
+Since the default for use-pull-ops is ``either``, normally pywbem first tries
+the pull operation and then if that fails, the traditional operation.  That
+is probably the most logical setting for ``use-pull-ops`` unless you are
+specifically testing the use of pull operations.
+
+
+.. _`pywbemcli commands to WBEM Operations`:
+
+pywbemcli commands to WBEM Operations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following table defines which pywbemcli commands are used for the
+corresponding pywbem request operations.
+
+=================================  ==============================================
+WBEM CIM-XML Operation             pywbemcli command-group & subcommand
+=================================  ==============================================
+**Instance Operations:**
+EnumerateInstances                 instance enumerate INSTANCENAME
+EnumerateInstanceNames             instance enumerate INSTANCENAME --name_only
```

Comment:
Update option names `--name_only` to `--names-only`. Occurs multiple times.

Reply:
DONE